packages/volto-slate/news/5818.bugfix

```diff
@@ -0,0 +1 @@
+Fix issue of duplicated blocks upon pasting an image into the Slate Editor. @aryan7081
```

Comment:
```suggestion
When pasting an image into a Slate block, now only an image block is created, instead of both a Slate block and an image block. @aryan7081
```